deploy/operator/hypershift/deploy_hypershift_cluster.sh

```diff
@@ -121,3 +121,35 @@ echo "Cluster installation has been stopped (either for good or bad reasons)"
 
 KUBECONFIG=$SPOKE_KUBECONFIG wait_for_condition "agentclusterinstall/${ASSISTED_AGENT_CLUSTER_INSTALL_NAME}" "Completed" "90m" "${SPOKE_NAMESPACE}"
 echo "Cluster has been installed successfully!"
+
+
+# Test webhooks
+
+function validate_webhook() {
+  log="$1"
+  pattern="$2"
+  success_msg="$3"
+  failure_msg="$4"
+
+  if [[ $log == *$pattern* ]];
+  then
+    echo $success_msg
+  else
+    echo $failure_msg
+    exit 1
+  fi
+}
+export -f validate_webhook
+
+echo "Test webhooks for InfraEnv successfull update"
+oc --kubeconfig $SPOKE_KUBECONFIG -n $SPOKE_NAMESPACE patch infraenv $ASSISTED_INFRAENV_NAME \
+    -p '{"metadata": {"annotations":{"foo":"bar"}}}' --type merge
+webhook_log=$(oc logs -n $SPOKE_NAMESPACE -l app=agentinstalladmission --since=2s | grep -E 'UPDATE(.)*infraenvs' | tail -1)
```

Comment:
2 seconds might be a bit of a stretch and cause delays if something gets delayed just a bit. no?
is it possible increasing to 10 seconds just to make sure?

Reply:
for my tests it should be quite enough, i'll increase to 5s just in case